python/common/document-record-service/src/document_record_service/constants.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+
+
+class DocumentClasses(Enum):
+    """Render an Enum of the document service document classes."""
+
+    COOP = "COOP"
+    CORP = "CORP"
+    SOCIETY = "SOCIETY"
+
+
+class DocumentTypes(Enum):
+    """Render an Enum of the document service document types."""
+
+    AFDV = "AFDV"
+    AMLG = "AMLG"
+    ANNR = "ANNR"
+    FRMA = "FRMA"
+    AMLO = "AMLO"
+    CNTO = "CNTO"
+    CNTI = "CNTI"
+    CNVS = "CNVS"
+    CORR = "CORR"
+    CRT = "CRT"
+    CORP = "CORP"
+    REGN = "REGN"
+    REGO = "REGO"
+    SYSR = "SYSR"
+
+
+# Map between filing and DRS document type
+# SYSR: alteration, appointReceiver, ceaseReceiver, changeOfDirectors,
+# incorporationApplication, restoration, noticeOfWithdrawal
+DOCUMENT_TYPES = {
```

Comment:
So, some document types will apply to different business types.  I can see that `DocumentClasses` is used to represent the groups of business types(corps, coop and firms).

Some questions around that:
- there are no firm(SP/GP) related entries.  think these will need to be added
- how do you represent documents that will be used across multiple document classes?  for example, annual reports will apply coops, firms and corps.